src/components/Modals/FiatRamps/fiat-ramps.md

```diff
@@ -11,6 +11,22 @@ Generally to add a new fiat ramp provider, you should complement `SupportedFiatR
 - The new provider should have a `onSubmit` property for passing the data to the provider API.
 - `isImplemeted` field is used to show whether the provider is ready to use or a `coming soon` text should be shown instead.
 
+You will also need to add a two-way mapping from ShapeShift AssetId <> ramp ticker / ID in the `@shapeshiftoss/caip` package, located in [lib](https://github.com/shapeshift/lib).
+Refer to the README in that repo to work with lib packages locally.
+
+This mapping serves two purposes:
+- have a source of truth for the intersection of the assets ShapeShift and the ramp support
+- be able to convert from our internal [`AssetId`](https://github.com/shapeshift/lib/tree/main/packages/caip#assetid-caip19---asset-type-and-asset-id-specification)s to the ramp's ticker / ID, so we can build the payload/link to the ramp
+
+See these commits for reference:
+- [Banxa mapping](https://github.com/shapeshift/lib/commit/f24f9d800041534ae45a5196bb2030bba5f5864a)
+- [Gem mapping](https://github.com/shapeshift/lib/commit/78a8f14b82330239555ad544121ea956dd6ca8be)
+- [Coinbase mapping](https://github.com/shapeshift/lib/commit/fb2cc5aafe74ac33d896f130952b4dcbfbf98e4a) (unused in [web](https://github.com/shapeshift/web))
+- [OnJuno mapping](https://github.com/shapeshift/lib/commit/5fade1f998cc6224dd2cb5d076f26a4c485b649a)
```

Comment:
Chose to name this one `OnJuno` vs. the `JunoPay` terminology we use, in case contributors decide to search for each of the ramps, makes Googling easier